src/index.js

```diff
@@ -0,0 +1,81 @@
+import './style.css';
+
```

Comment:
- [ ] Kindly for this milestone you should have an array of simple data on your index.js and a function that display this data. Kindly add them as asked by the requirements.

![image](https://user-images.githubusercontent.com/35535130/167507874-49759e9d-b89a-4a80-a4a0-ee013803c18d.png)

Reply:
Done